package.json

```diff
@@ -7,7 +7,7 @@
     "test": "npm run test:lint src && npm run test:deps",
     "test:lint": "standard src",
     "test:deps": "dependency-check --verbose --missing ./src/**/*.js ./src/scripts/index.js --no-dev --i mysql2 --i dotenv-safe --i sequelize-cli",
-    "dev": "babel-node -r ./node_modules/dotenv-safe/config ./src/server.js",
+    "dev": "babel-watch ./src/server.js",
```

Comment:
Could this be applied to `tracks-api` as well to make it reload on changes? I can put in that PR if so.

Reply:
Yes, just make sure to not load dotenv-safe twice while in production mode. babel-watch does not support the require option.